example/src/main/java/com/stripe/example/activity/LaunchPaymentSheetCustomActivity.kt

```diff
@@ -72,15 +72,6 @@ internal class LaunchPaymentSheetCustomActivity : BasePaymentSheetActivity() {
                 0
             )
             viewBinding.buyButton.isEnabled = true
-        } else {
```

Comment:
Why is it that we can remove this now?

Reply:
We don't want to set the drawable on failure or cancel, we should just leave whatever value was already present.